scripts/tasks.py

```diff
@@ -0,0 +1,46 @@
+import os
+from pathlib import Path
+
+from invoke import task
```

Comment:
We've not yet used `invoke` in this project. Not sure whether we should.

@woile I guess this might not be what we want? (This is something similar to makefile I personally like to use. If my memory serves me right, we don't want to introduce such thing in commitizen?)

Reply:
ok !
I made a change in
 [ci(scripts): a script to gen cli help screenshots](https://github.com/commitizen-tools/commitizen/pull/1048/commits/b1dc3f721bfa9061d25e09904c9ee50773606093)
 
 Please check it, thank you ~~